lib/logger.js

```diff
@@ -13,7 +13,7 @@ const hostname        = tools.getHostName();
 let SysLog;
 
 try {
-    SysLog = require('winston-syslog').Syslog;
+    SysLog = require(__dirname + '/winston-syslog.js').Syslog;
```

Comment:
Bitte nicht `__dirname` in require verwenden. Ein relativer Pfad (`./`) ist besser:
```
SysLog = require('./winston-syslog').Syslog;
```

Reply:
Hatte ich auch zuerst. aber oben hat jemand das mit __dirname für tools.js verwendet ;-)
Aber, kein Problem mit ./

Reply:
Dann hab ich das noch nicht geändert ;)